### DISCUSSION TITLE
Showcase CRUD : checkboxes have disappeared

### DISCUSSION THREAD

**Discussion by ritonglue** [primefaces/community#3174] [PrimeFaces] [answered]
Starting with 14.0.0, to enable selection with checkboxes or radiobuttons, you have to set `selectionBox` to true.
As a consequence, the crud example has lost its checkboxes and probably other facelet too.
https://www.primefaces.org/showcase/ui/data/datatable/crud.xhtml

I think it would be better to fix these pages and set the selectionBox to true.

**Comment by melloware** [comment]
Its fixed here which is the 15.0.0 Showcase: https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/data/datatable/crud.xhtml

**Reply by melloware** [reply, marked as answer]
OK PR submitted: https://github.com/primefaces/primefaces/pull/13037/files

**Reply by AndrewChurchill1970** [reply]
is it possible to get a fix for version 14 ? we also have implemented <p:column selectionBox="true" ...

**Reply by melloware** [reply]
@AndrewChurchill1970 you can see the fix right here: https://quarkus-faces-melloware-8a6a34c1.koyeb.app/ui/data/datatable/crud.xhtml  Its the same as 14.X
